main.py

```diff
@@ -26,19 +26,18 @@ def create_connection(db_file):
 
 def setup_db():
     c = conn.cursor()
-    c.execute('''CREATE TABLE IF NOT EXISTS votes (id INTEGER AUTO_INCREMENT PRIMARY KEY, user_id INTEGER, votes INTEGER)''')
-    c.execute('''CREATE TABLE IF NOT EXISTS votes_history (id INTEGER AUTO_INCREMENT PRIMARY KEY, user_id INTEGER, message_id INTEGER, backer INTEGER, timestamp DATETIME DEFAULT CURRENT_TIMESTAMP)''')
+    c.execute('''CREATE TABLE IF NOT EXISTS votes_history (id INTEGER AUTO_INCREMENT PRIMARY KEY, user_id INTEGER, message_id INTEGER, backer INTEGER, vote_time DATETIME DEFAULT CURRENT_TIMESTAMP)''')
 
-def processVote(author, message_id, backer):
-    row = getUser(conn, author)
+def processVote(message_id, author, backer):
+    if updateHistory(conn, author, message_id, backer):
+        print(f"{author} got an upvote!")
 
-    if row == None:
-        createUser(conn, author)
-    else:
-        updateVote(conn, author, row[1]+1)
-
-    updateHistory(conn, author, message_id, backer)
-    print(f"{author} got an upvote!")
+def processDownvote(message_id, author = -1, backer = -1):
```

Comment:
Like in the first comment: Is there a Particular difference between None and -1? Wouldn't it be better if we were checking for None's instead